internal/common/auth/service_wrapper.go

```diff
@@ -30,7 +31,8 @@ import (
 )
 
 const (
-	jwtHeaderName = "cadence-authorization"
+	jwtHeaderName     = "cadence-authorization"
+	JWTInternalIssuer = "internal-jwt"
```

Comment:
I propose separating internal constants from external.

https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations

Reply:
thanks, moving this const to another package, this doesn't belong here.